### PR TITLE
Film advertising

### LINKED ISSUE #1: Create box ''filmreklam'' for every page
Create box with image or video (iframe for example) for movie advertising.
Add Title

**_User Story:
As a user i want to see some advertising for the upcoming movies on the website._**

### PR DESCRIPTION
**Pull request by MikaelaMathisson**
Added the boxes of film advertising to index.html and news.html.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,7 +19,19 @@
...
+      <iframe
+        width="1000"
+        height="315"
+        src="https://www.youtube.com/embed/DCtjeTmJhYs?si=cbZ63StJYYydfSJU&amp;controls=0"
+        title="YouTube video player"
+        frameborder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+        allowfullscreen
+      ></iframe>
+      <h2>Hammarsköljd</h2>
```
Comment (sebastianpiresmolin) [inline index.html]: Hammarskjöld*

Comment (sebastianpiresmolin) [inline index.html]: Maybe add transform(y) in css so that the text ''overlays'' the iframe? I think that would look nice!

**Review by sebastianpiresmolin — DISMISSED** [review]

File: index.html (hunk 1 above)
Comment (MikaelaMathisson) [reply index.html]: Nice catch! Will change this when I can

Comment (MikaelaMathisson) [reply index.html]: Good idea! I Will try to do that. If I have any problems to do it, I would love to get your help. 👍🏽 I will let you know

**Comment by MikaelaMathisson** [comment]
@sebastianpiresmolin I fixed the typo and added the translate y. 
I did the translate y to make the title of the movie go up in the iframe but i couldnt fit the premiere date without it interfering with the iframe. So i put it under. I think it looks fine. 
If you have time, please check this and test.

**Pull request merged**